Source/JobDriver_ZombieHunt.cs

```diff
@@ -43,31 +65,6 @@ protected override IEnumerable<Toil> MakeNewToils()
             }).FailOnDespawnedOrNull(TargetIndex.A);
         }
 
-        public override void Notify_PatherFailed()
```

Comment:
I dont see how they still attack doors after removing this. let me know pls

Reply:
I guess JobDriver already implements this? It is working but I dont understand why

Reply:
🤔 I didn't test it, but I will certainly re-add the door feature if this broke it, now that I understand how it works.

I think I'll have to re-work it a little bit but that shouldn't be too bad.

Reply:
It still works thats why Im confused. You removed it but it still works *shrugs*

Reply:
This functionality is copied from manhunting animals so Im not completely sure what is going on. I assume this block of code is only for some specific scenarios so it is not immediately noticeable. So I would just keep it

Reply:
Well, by default the `FollowAndMeleeAttack` toil uses the pawn's pather and that uses the mode that allows passing through doors, so it doesn't consider it a pather failure when the door is in the way. Manhunting animals might use a different pather by default that doesn't consider doors okay. 🤔